src/shared/logger.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Google LLC

```

Comment:
First off, thank you for creating a separate `logger.js` file!
The `header-check` is failing before the year should be `2023` on new files introduced by this pull-request.
```suggestion
 * Copyright 2023 Google LLC

```

Reply:
I've resolved this one